### ISSUE TITLE
can't hide objects in scene

### ISSUE THREAD

**Issue by Packsod** [Gatada/MultiResolutionCameras#1]
Hi,Gatada，
two bugs I found in jb-multicamera.py 
the previous version did not have this error.
the new version of your multi resolution camera addon cause Blender unable to hide objects, neither use shotcut nor clicking the eye icon in the outline panel.
another one is the checkbox “highlight selects Camera” ，turn it on will make Blender freezes,
tested it in Blender2.41 and Blender2.61LTS ,without any other addons,

**Comment by Gatada** [comment]
Hey, do you mean 3.61? Cause the most recent update requires 3.61.

**Comment by Gatada** [comment]
Thanks for the heads-up, I found and fixed the issue relating to hiding objects. Looking into the second bug now.

**Comment by Gatada** [comment]
Found the recursive call causing the hanging and fixed it. At least in my testing, now everything works as expected. Thanks @Packsod for letting me know!

**Comment by Gatada** [comment]
This is what I get when trying to do something quickly when I don't have time: bugs.

Not sure what I did wrong, but now previewing camera animation sequences no longer works. So I'll have to look at this and related bugs later today.

**Comment by Packsod** [comment]
got it fixed now, thanks for your work,
I don't use animation sequences in my workflow, so I'm afraid I can't give you any advice,
but I found a small new problem
“refresh camera list" button in the addon only appears once when setting up the scene before I click it.
after that ,camera list will not be automatically refreshed when creating, deleting or modifying the name of the cameras.
 to make it work temporarily, I have saved the “refresh camera list" button to my Quick Favorites（hotkey Q）,it works.
